### DISCUSSION TITLE
Tools in Linux to batch update "Part Of Compilation" tag for fix broken up into lots of separate albums issue

### DISCUSSION THREAD

**Discussion by halida** [navidrome/navidrome#2253] [General]
According to:
https://www.navidrome.org/docs/faq/#i-have-an-album-with-tracks-by-different-artists-why-is-it-broken-up-into-lots-of-separate-albums-each-with-their-own-artist

But I did not find any tool can update TCMP tag, checked those:
https://www.baeldung.com/linux/editing-mp3-tags

Any idea?

**Comment by halida** [comment]
Found one: sudo apt install kid3

**Comment by deluan** [comment]
If you want command line tools, you may also want to take a look at https://beets.io/, an excellent media organization tool.

**Reply by mcrapet** [reply]
I use beets I do have a clean database but I disabled tag-rewrite and copy features on purpose. So I don't have the neat music directory tree that navidrome can import.

Is there any way that navidrome considers using my beets database (which is SQLite) as import/update?
Note: the database contains path to the actual music files.

**Comment by ElHado** [comment]
you can use eyeD3 command line:
`eyeD3 --text-frame TCMP:1 *.mp3`
